report/4.Design/Performance.tex

```diff
@@ -0,0 +1,45 @@
+\section{Performance}
+In order to be able to track an object in real time, the object localization will have to happen in real time.
+Therefore, the algorithms should be tested in order to determine whether they will be able to execute the object localization fast enough for it to be feasible for the project.
+
+\subsection{Raspberry Pi}
+The Raspberry Pi 3B is the desired host of the MI processing, but it is very limited in the amount of resources available to it.
+It comes with a 1.2GHz Quadcore ARM Cortex-A53, 64Bit processor and 1 GB of RAM.
```

Comment:
Source (ref til https://www.raspberrypi.org?)

Reply:
Or maybe a cross reference to the introduction where we discuss the hardware platforms - I think we have the same information there

Reply:
Har tilføjet backref